nautobot/extras/filters/__init__.py

```diff
@@ -979,6 +980,21 @@ class Meta:
         fields = "__all__"
 
 
+class ObjectMetadataFilterSet(NautobotFilterSet):
+    q = SearchFilter(
+        filter_predicates={
+            "value": "icontains",
+            "metadata_type__name": "icontains",
+            "contact__name": "icontains",
+            "team__name": "icontains",
+        },
+    )
+
+    class Meta:
+        model = ObjectMetadata
+        fields = "__all__"
```

Comment:
Does this result in an undesired `_value` filter? Do we need to implement a custom `value` filter now?

Reply:
Sounds like we need to implement a similar mixin like `CustomFieldModelFilterSetMixin`?

Reply:
Maybe longer term (so we can filter an object by its metadata) but I'm not sure I follow how that's necessary to address this specific concern?